src/com/severalcircles/colors/commands/CommandUnknown.java

```diff
@@ -0,0 +1,105 @@
+                break;
+            case 7:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.JUMP, 2400, 50));
+                break;
+            case 8:
+                player.getWorld().spawnEntity(player.getLocation(), EntityType.PRIMED_TNT);
+                player.addPotionEffect(new PotionEffect(PotionEffectType.SLOW, 80, 100));
+                break;
+            case 9:
+                for (int i = 0; i < 10; i++) {
+                    player.getWorld().spawnEntity(player.getLocation(), EntityType.ZOMBIE);
+                }
+                player.getWorld().spawnEntity(player.getLocation(), EntityType.GIANT);
+                player.addPotionEffect(new PotionEffect(PotionEffectType.WEAKNESS, 1500, 10));
+                break;
+            case 10:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.FIRE_RESISTANCE, 2400, 1));
+                break;
+            case 11:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.GLOWING, 3600, 1));
+                break;
+            case 12:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.WITHER, 1000, 1));
+                break;
+            case 13:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.POISON, 1200, 1));
+                player.getWorld().spawnEntity(player.getLocation(), EntityType.BAT);
+                break;
+            case 14:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.INVISIBILITY, 3600, 1));
+                break;
+            case 15:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.WATER_BREATHING, 4800, 1));
+                player.addPotionEffect(new PotionEffect(PotionEffectType.DOLPHINS_GRACE, 4800, 1));
+                break;
+            case 16:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.ABSORPTION, 1200, 5));
+                break;
+            case 17:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.HUNGER, 1200, 10));
+                break;
+            case 18:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.INCREASE_DAMAGE, 500, 100));
+                Bukkit.broadcastMessage(ChatColor.DARK_RED + "ACTIVATE INSTANT KILL MODE");
+                break;
+            case 19:
+                for (int i = 0; i < 420; i++) {
+                    player.getWorld().spawnEntity(player.getLocation(), EntityType.EXPERIENCE_ORB);
+                }
+                player.getWorld().spawnEntity(player.getLocation(), EntityType.PIG);
+                player.sendMessage(ChatColor.YELLOW + "You've been greeted by the good luck pig");
+                break;
+            case 20:
+                player.addPotionEffect(new PotionEffect(PotionEffectType.HERO_OF_THE_VILLAGE, 48000, 1));
+                break;
+
+        }
+        return true;
+    }
+    }
+//hehe funny command
```

Comment:
Looks like somebody forgot to clean up comments

Reply:
no i just added that